pkg/controller/flink/container_utils.go

```diff
@@ -60,6 +61,13 @@ func getCommonAnnotations(app *v1beta1.FlinkApplication) map[string]string {
 	return annotations
 }
 
+func GetPrometheusAnnotations(port int) map[string]string {
+	return map[string]string{
+		"prometheus.io/scrape": "true",
```

Comment:
Shouldn't this be configurable too? I wouldn't return this annotation if `prometheus.io/scrape` is `false`.

Reply:
I'm going to and an optional configuration option called `enableMetrics` that has a default value of `false`. When it's false, no Prometheus-related annotations will be added at all.

Reply:
I'm wondering if it would be simpler to just have all annotations be explicitly listed on the configuration file (see my comment below on generic annotations). If you do this, then this would be extensible to any type of annotation, and not Prometheus-specific. Something like (potentially separating annotations for JM and TM):

```
operator:
   annotations:
      job_manager:
         prometheus.io/scrape: true
         prometheus.io/port: 9249
         foo: bar1
      task_manager:
         prometheus.io/scrape: true
         prometheus.io/port: 9249
         foo: bar2
```

Reply:
Actually, this seems related to the discussion going on in #10 (annotations on the ingress resource). Cc: @anandswaminathan

Reply:
You can pass the annotations from the flink application right, like here - https://github.com/lyft/flinkk8soperator/blob/master/examples/wordcount/flink-operator-custom-resource.yaml#L6

Reply:
Oh I see. I totally missed that the user can configure annotations at the CR level. I'll close this PR then.